src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py

```diff
@@ -28,12 +27,18 @@
       - setJobParameter()
       - deleteJobParameters()
 """
-import hashlib
-
+from xmlrpc.client import boolean
```

Comment:
?

Reply:
Sometimes VScode automatically adds random imports from what I write :/

Reply:
`git add --patch` is a beautiful way to see these things and not commit them.